src/helpers/figures/charts/line_chart.ts

```diff
@@ -209,6 +212,56 @@ export class LineChart extends AbstractChart {
     );
     return new LineChart(definition, sheetId, this.getters);
   }
+
+  highlightItem(index: number, dataSets) {
+    dataSets.forEach((dataset) => {
+      const backgroundColors = dataset.backgroundColor;
+      if (!backgroundColors) {
+        return;
+      }
+      backgroundColors.forEach((color, i, colors) => {
+        colors[i] = setColorAlpha(color, i === index ? 1 : 0.5);
+      });
+    });
+  }
+
+  unHighlightItems(dataSets) {
+    dataSets.forEach((dataset) => {
+      const backgroundColors = dataset.backgroundColor;
+      backgroundColors.forEach((color, i, colors) => {
+        colors[i] = setColorAlpha(color, 1);
```

Comment:
maybe use lightenColor rather than setting the alpha, so the behavior is the same for all chart backgroud colors